fe1-web/src/features/social/reducer/SocialReducer.ts

```diff
@@ -27,7 +26,7 @@ interface SocialReducerState {
 }
 
 // Root state for the Social Reducer
-interface SocialLaoReducerState {
+export interface SocialLaoReducerState {
```

Comment:
why do we need to export this? 🤔 I'm sure there's a good reason and I've just missed something.

Reply:
It is to explicitly define the type of the reducer in the [`social/interface/Configuration.ts`](https://github.com/dedis/popstellar/blob/work-fe1-kschneiter-inject-dependencies-social/fe1-web/src/features/social/interface/Configuration.ts) file (line 90).